bot.py

```diff
@@ -212,4 +240,11 @@ async def message_not_modified_handler(update, error):
 
 
 if __name__ == "__main__":
+    print("Launching Timeout Worker")
+    loop = asyncio.get_event_loop()
+    asyncio.run_coroutine_threadsafe(timer.notify(), loop)
```

Comment:
Here is how it should be running co-routines together. 

I ran the lottery but the bot didn't respond with anything when it finished while I see that it should send either winning message or not-winning anyone message.

Reply:
It works! Thank you!

"I ran the lottery but the bot didn't respond with anything when it finished while I see that it should send either winning message or not-winning anyone message."

It works for me when I tested it right now